src/ButtonComponent/Button.tsx

```diff
@@ -0,0 +1,14 @@
+import cn from 'classnames';
+import React from 'react';
+
+import { ButtonProps } from './types';
+
+const Button = ({ content }: ButtonProps): JSX.Element => {
```

Comment:
I think using `children` would be better instead of creating a `content` property.

Reply:
i will consider that in the next commit